pkcs11/_pkcs11.pyx

```diff
@@ -1162,14 +1166,14 @@ cdef class lib:
     cdef public tuple library_version
 
     def __cinit__(self):
-        assertRV(C_Initialize(NULL))
+        assertRV(C_GetFunctionList(&_funclist))
```

Comment:
The one other thing more we could do is retrieve `G_GetFunctionList` specifically from `dlopen` rather than binding the whole symbol list in like it does today. This would make it much, much easier to port to Windows, which can't `dlopen` all window at once.

Reply:
This is addressed now, through the call to `_load_pkcs11_lib`. BTW I had to move the library loading from `_loader.pyx` into `_pkcs11.pyx`to avoid sharing another global between modules. The loading of the PKCS#11 library as well as the function binding is confined within this method.